tests/test_submit_artifacts.py

```diff
@@ -427,44 +427,45 @@ def test_setup_scan_prereqs(self):
                     submit_artifacts.setup_scan_prereqs(upload_api=upload_api)
                 )
 
-        # Build in progress, abort new build
+        # Build in progress, try to cancel and recreate
         with patch(
             "veracode.submit_artifacts.is_ready_for_new_build", return_value=False
         ):
-            self.assertFalse(submit_artifacts.setup_scan_prereqs(upload_api=upload_api))
+            # Unsuccessful create build
```

Comment:
Confusing comment, suggest removing.
```suggestion
```

Reply:
Missed this the first time around, fixed now.